python/earthquake-tracker/earthquake_tracker/main.py

```diff
@@ -19,6 +19,7 @@
 # Then, let's write a function that queries the USGS for information on recent earthquakes.
 # Our function will take in a time range and return the id, place, magnitude, and timestamp
 # of all earthquakes that occured in that time range.
+# We annotate this function with `@DBOS.step` so we can durably call it from a workflow later on.
```

Comment:
"durably call" is a short thing to say, but I'm not sure it's clear.

Reply:
(applies several places below)

Reply:
Will keep iterating on this, but I think it's best to briefly mention it here and to explain it in detail around the actual workflow.

Reply:
"Call it from a durable workflow" instead of "durably call it from a workflow"?
or
"# We annotate this function with `@DBOS.step` so that workflows will record its output and not repeat it on recovery."
?

Reply:
Good call, I like "call it from a durable workflow" way better